### PR TITLE
Add notes on deployment and model explainability

### PR DESCRIPTION
**Pull request by calderonjesus**
+ Adds note on model explainability methods
+ Adds notes on deployment.
+ Update experiment code and notes

### CODE REVIEW

File: src/db/.env [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #Postgres
-POSTGRES_USER=postgres
+POSTGRES_USER=posgres
```
Comment (danielrazavi) [inline src/db/.env:2]: is this correct?

**Review by danielrazavi — CHANGES_REQUESTED** [review]
No comments in your python code, please add some comments for better readability.

File: src/db/.env (hunk 1 above)
Comment (calderonjesus) [reply src/db/.env:2]: No, thanks for highlighting. Added the t back in.

**Comment by danielrazavi** [comment]
> Thank you, @danielrazavi. I'll add docstrings to functions and comments where relevant in the next iteration.

Would the new changes be in dev0219? Should I close this PR and await changes there?

File: src/db/.env (hunk 1 above)
Comment (calderonjesus) [reply src/db/.env]: Done.

**Comment by calderonjesus** [comment]
Added docstrings.

File: README.md [hunk 2]
```diff
@@ -40,27 +54,38 @@ By the end of this course, a student will be able to:
 
 + Instructor: Jesús Calderón (he/him)
 
-  * Email etiquette
+  * dsi.production.course [at] gmail.com
```
Comment (danielrazavi) [inline README.md:57]: is this email address actually live? 😯

Comment (calderonjesus) [reply README.md:57]: yes

**Review by danielrazavi — APPROVED** [review]

**Pull request merged**